.travis.yml

```diff
@@ -23,3 +23,6 @@ install:
 
 script:
   - ./gradlew build -PcheckJava6Compatibility --daemon -s -i && ./gradlew ciPerformRelease
+
+after_success:
+  - ./gradlew upgradeDownstream
```

Comment:
The thing is that "assertReleaseNeeded" is ran with "ciPerformRelease" too. So if it fails, everything is fine, it would fail again in "after_success" ("after_success" is run because "ciperformRelease" is set to stop execution if "assertReleaseNeeded" fails, without failing the build). But if it passes, release would be executed and the version changed and pushed. So if we check again in "after_success" we would fail on "assertReleaseNeeded" AFAIK. Maybe we can somehow pass information between executed tasks? Maybe through creating a file in the workspace, with information if "assertReleaseNeeded" failed or not?